src/cache/stub.go

```diff
@@ -7,9 +7,14 @@ package cache
 
 import (
 	"fmt"
+
 	"github.com/thought-machine/please/src/core"
 )
 
 func newRPCCache(config *core.Configuration) (*httpCache, error) {
 	return nil, fmt.Errorf("Config specifies RPC cache but it is not compiled")
 }
+
+func newRemoteCache(state *core.BuildState) core.Cache {
```

Comment:
Why does this file exist - can we not just compile in the remote caches for every use?

Reply:
Just wanted to avoid the dependencies during bootstrap - it adds a bunch of extra Go modules that have to be downloaded & compiled but never actually used.